--- tests/test_csimulator.py
+++ tests/test_csimulator.py
@@ -60,11 +60,11 @@ def test_simulator_setter_success():

     csim.model = MockModel()

-    with (
-        unittest.mock.patch("os.path.abspath", side_effect=lambda x: x),
-        unittest.mock.patch(
-            "bionetgen.simulator.csimulator.CSimWrapper"
-        ) as mock_wrapper,
-    ):
+    with unittest.mock.patch(
+        "os.path.abspath", side_effect=lambda x: x
+    ), unittest.mock.patch(
+        "bionetgen.simulator.csimulator.CSimWrapper"
+    ) as mock_wrapper:
         csim.simulator = "dummy_lib_file"
         mock_wrapper.assert_called_once()
